src/main/java/org/example/patterns/Tower_of_hanoi.java

```diff
@@ -0,0 +1,21 @@
+import java.io.*;

```

Comment:
- Rename class and file to _TowerOfHanoi_ (CamelCase)
- Package info at top of file is missing

Please see other classes for reference.